src/app/pages/home/home-page.component.spec.ts

```diff
@@ -23,6 +24,13 @@ describe('Pages: Home page', () => {
 					CommonModule,
 					HttpClientModule,
 					StoreModule.forRoot({}),
+					StoreModule.forFeature(
```

Comment:
forRoot and forFeature in the same spec? Are both necessary?

Reply:
When testing with forFeature functionality, the reducerManager isn't provided, which is when you add forRoot